src/gui/guiFormSpecMenu.cpp

```diff
@@ -2794,6 +2795,16 @@ void GUIFormSpecMenu::parseModel(parserData *data, const std::string &element)
 	e->enableContinuousRotation(inf_rotation);
 	e->enableMouseControl(mousectrl);
 
+	s32 frame_loop_begin = 0;
```

Comment:
Are these the defaults in the mesh? (Because we're setting these unconditionally on the mesh of the no frameloop was specified.)

Reply:
As far as I understood the Irrlicht docs (http://irrlicht.sourceforge.net/docu/classirr_1_1scene_1_1_i_animated_mesh_scene_node.html#a900400fe375ca13f48876b84900ffddf), 0 is always the default for the begin and the "MaximalFrameCount" is the default end. As the latter one isn't readily available here, I used the maximum s32 number. The clamping is done by Irrlicht: https://github.com/zaki/irrlicht/blob/040aa5043af8725e3b201b9fca3c0536fd298d33/source/Irrlicht/CAnimatedMeshSceneNode.cpp#L494

So, according to the Irrlicht docs and source code, and according to my testing, these default values that are used here should have no unintended influence.

Thank you very much for your review!

Reply:
Cool.